src/clusterfuzz/_internal/tests/appengine/handlers/cron/cleanup_test.py

```diff
@@ -1363,7 +1363,7 @@ def test_top_crashes_match_and_label_removed(self):
 
 @test_utils.with_cloud_emulators('datastore')
 class UpdateComponentsTest(unittest.TestCase):
```

Comment:
Let's add a test that checks `component_id` is assigned.

Reply:
I think that's the most I can do about it, because otherwise it would require tons of work to rework how the cleanup.py and cleanup_test.py work. Those are really tied to monorail initially, and aren't really made to have per-tracker cases. I think that would be too much work just for that feature.

Reply:
Good enough, thanks!